lua/lspconfig/vdmj.lua

```diff
@@ -0,0 +1,127 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+if vim.fn.has 'nvim-0.5.1' == 0 then
+  vim.notify('The VDMJ language server requires nvim > 0.5', vim.log.levels.ERROR)
+  return
+end
+
+local server_name = 'vdmj'
+
+local mavenrepo = util.path.join(vim.env.HOME, '/.m2/repository/com/fujitsu')
```

Comment:
let's not do '/.m2/repository/com/fujitsu', but split this up into individual arguments to path join